docs/getting-started-tutorial/1-view-first-development.adoc

```diff
@@ -0,0 +1,71 @@
+:idprefix:
+:idseparator: -
+:toc: right
+:toclevels: 2
+
+# View-first Development
+
+If you're developing a user-facing web site or application, one of Lift's
+greatest improvements over existing systems is view-first development.
+View-first development thoroughly separates the process of creating the user
+interface from the process of putting data from the system into it, in a way
+that lets you stay focused on users when you're creating the user interface and
+worry about the the interface between your backend and the HTML only when
+you're working on the backend.
+
+The flip side of view-first development is that it takes some getting used to
+if one is accustomed to the typical web MVC framework. The first stop when
```

Comment:
`if one is accustomed` — I think we should feel free to use the second-person pronouns where applicable. We do so in the first paragraph, and it makes the sentence feel more personal and conversational to me to read `if you're accustomed`. Using `one` has more of an academic, aloof texture.

Reply:
I prefer no contractions, since this is written, but I'm down to go second-person to make it less academic.